Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM golang:alpine
 WORKDIR /usr/src/app
 
 RUN adduser -u 9000 -D app
-ADD build/codeclimate-gofmt /usr/src/app
+COPY . /usr/src/app
```

Comment:
What does this change do to the size of the image? Did you try doing `COPY build/codeclimate-gofmt /usr/src/app` instead of `ADD`?